app/views/sessions/_card.html.erb

```diff
@@ -71,7 +71,7 @@
   <% end %>
 
   <div class="flex flex-col">
-    <% if session.tags.exists? %>
+    <% if session.tags.present? %>
```

Comment:
Similar to the `#count` changes, `#exists?` will execute a query whereas `#present?` will load records if not loaded already and delegate to them. In this case they have already been pre-loaded.